riscv-system-description.tex

```diff
@@ -69,6 +69,9 @@
 \label{index}
 \printindex
 
+\chapter{External Industrial Standards}
```

Comment:
I would like to add this section for external industrial standards, such as DT, SMBIOS, ACPI and etc. that would be changed for the corresponding system description introduced in this spec.